bokeh/command/bootstrap.py

```diff
@@ -0,0 +1,32 @@
+'''
+
+'''
+import argparse
+
+from bokeh.settings import settings
+
+from .subcommands.html import HTML
+from .subcommands.json import JSON
+from .subcommands.serve import Serve
+
+subcommands = [Serve, HTML, JSON]
+
+def main(argv):
+    parser = argparse.ArgumentParser(prog=argv[0])
+
+    # does this get set by anything other than BOKEH_VERSION env var?
+    version = settings.version()
+    if not version:
+        version = "unknown version"
+    parser.add_argument('-v', '--version', action='version', version=version)
+
+    subs = parser.add_subparsers(help="Sub-commands")
+
+    for cls in subcommands:
+        subparser = subs.add_parser(cls.name, help=cls.help)
+        subcommand = cls(parser=subparser)
+        subparser.set_defaults(func=subcommand.func)
+
+    # TODO (bev) unhelpful error if no subcommand given
```

Comment:
I get:
```
usage: bin/bokeh [-h] [-v] {serve,html} ...
bin/bokeh: error: too few arguments
```
reading over the argparse docs, the only hack I see here is to subtype ArgumentParser and override the `error` method which one that replaces this exact string if it gets this string. It will only work in English (which is probably for the best, since our replacement won't be translated).

Another possibility would be to make the parser accept no subcommand and then generate an error ourselves, but then the usage summary would change and show that a missing subcommand is allowed I think.